_data/simple-icons.json

```diff
@@ -10279,6 +10279,11 @@
             "hex": "EC1C24",
             "source": "https://en.wikipedia.org/wiki/File:Shanghai_Metro_Full_Logo.svg"
         },
+        {
+            "title": "ShareX",
+            "hex": "2885F1",
+            "source": "https://getsharex.com/img/ShareX_Logo.svg"
```

Comment:
Where is this logo when you visit the site? Ideally, our sources should be webpages or git repos where we can check generally for any updates.

Reply:
Boom - let's set this URL as our source then!